test/codegen/swift/__run_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
You ran this, right? It's easy to forget as this is not hooked in the testing / release scripts.

Reply:
Yes, I have been using them as the primary way to test.